Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -14,6 +15,7 @@ public interface IGitModule
         IList<IGitRef> GetRefs(bool tags = true, bool branches = true);
         IEnumerable<string> GetSettings(string setting);
         IEnumerable<IGitItem> GetTree(string id, bool full);
```

Comment:
❓ Is this supposed to be marked `[Obsolete]`?

Reply:
Fixed.